largescaleobserved.qmd

```diff
@@ -4,6 +4,8 @@ fig-height: 3
 fig-dpi: 192
 fig-format: png
 engine: julia
+execute:
+  cache: true
```

Comment:
ideally we should be able to set this at the project level and not need to specify it in every file

[noblock]